flutter-studio/src/io/flutter/android/AndroidModuleLibraryManager.java

```diff
@@ -257,7 +257,7 @@ public void syncSkipped(@NotNull Project project) {
     };
 
     GradleSyncInvoker.Request request = new GradleSyncInvoker.Request(TRIGGER_PROJECT_MODIFIED);
-    request.runInBackground = true;
+    //request.runInBackground = true;
```

Comment:
Do you intend to keep this?

Reply:
Yes, I *think* I need to add the ability to remove the comment for all AS builds other than canary.